service/history/tasks/activity_retry_timer.go

```diff
@@ -41,6 +41,7 @@ type (
 		EventID             int64
 		Version             int64
 		Attempt             int32
+		Stamp               int32
```

Comment:
You need to add [serializer](https://github.com/temporalio/temporal/blob/d34367b20235490a632b49a0f096114bf1634fa5/common/persistence/serialization/task_serializer.go#L721)/deserializer also.

Reply:
probably done...please check if anything else is needed.